app/components/publishers/vacancies_component.rb

```diff
@@ -53,7 +53,7 @@ def view_applicants(vacancy, card)
                            organisation_job_job_applications_path(vacancy.id),
                            class: "govuk-link--no-visited-state")
       tag.div(card.labelled_item(I18n.t("jobs.manage.applications"), link))
-    elsif vacancy.job_applications.withdrawn.any? || vacancy.job_applications.none?
+    elsif vacancy.job_applications.withdrawn.any? || vacancy.job_applications.draft.any? || vacancy.job_applications.none?
```

Comment:
It's only the count that's different isn't it? Could we rewrite the method to make it a bit simpler?

Reply:
I'm confused by it - why isn't it equivalent to `else`?

Reply:
code is simplified now